src/plugins/blocksmith/components/divider-block-modal/index.jsx

```diff
@@ -0,0 +1,198 @@
+import moment from 'moment';
+import PropTypes from 'prop-types';
+import React from 'react';
+
+import Message from '@gdbots/pbj/Message';
+import {
+  Button,
+  Checkbox,
+  FormGroup,
+  Input,
+  Label,
+  Modal,
+  ModalBody,
+  ModalFooter,
+  ModalHeader, Select,
+} from '@triniti/admin-ui-plugin/components';
+import DividerBlockPreview from '@triniti/cms/plugins/blocksmith/components/divider-block-preview';
+import DateTimePicker from '@triniti/cms/plugins/blocksmith/components/date-time-picker';
+
+import changedDate from '../../utils/changedDate';
+import changedTime from '../../utils/changedTime';
+import PicklistPicker from '../../../sys/components/picklist-picker';
+
+class DividerBlockModal extends React.Component {
+  static propTypes = {
+    block: PropTypes.instanceOf(Message).isRequired,
+    isFreshBlock: PropTypes.bool.isRequired,
+    isOpen: PropTypes.bool,
+    onAddBlock: PropTypes.func.isRequired,
+    onEditBlock: PropTypes.func.isRequired,
+    toggle: PropTypes.func.isRequired,
+  };
+
+  static defaultProps = {
+    isOpen: false,
+  };
+
+  constructor(props) {
+    super(props);
+    const { block } = props;
+
+    this.state = {
+      hasUpdatedDate: block.has('updated_date'),
+      text: block.get('text') || '',
+      strokeColor: block.get('stroke_color'),
+      strokeStyle: block.get('stroke_style', 'solid'),
+      updatedDate: block.has('updated_date') ? moment(block.get('updated_date')) : moment(),
+      isValid: true,
```

Comment:
@cohencodes this doesnt do anything? can probably remove it entirely, unless you want to implement some validation on the text